pkg/recordset/recordset.go

```diff
@@ -220,31 +288,44 @@ func (m *Manager) createMissingTargetStacks(sourceStacks, targetStacks []cloudfo
 			m.logger.Log("level", "debug", "message", fmt.Sprintf("data for %q: %#v", sourceClusterName, data))
 			if err != nil {
 				m.logger.Log("level", "error", "message", fmt.Sprintf("could not get data about %q: %v", sourceClusterName, err))
+				continue
 			}
 
 			input, err := m.getCreateStackInput(targetStackName, data, source)
 			if err != nil {
 				m.logger.Log("level", "error", "message", fmt.Sprintf("could not create target stack input %q: %v", targetStackName, err))
+				continue
 			}
 
 			_, err = m.targetClient.CreateStack(input)
 			if err != nil {
 				m.logger.Log("level", "error", "message", fmt.Sprintf("could not create target stack %q: %v", targetStackName, err))
```

Comment:
The logging in general is a bit off. See https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#error-handling.

Reply:
improved overall logging, PTAL